mlir/lib/Support/Passes.cpp

```
@@ -41,25 +43,66 @@ runWithPassManager(ModuleOp module,
+                               "Run the default MQT QCO optimization pipeline.",
+                               populateDefaultQCOOptimizationPipeline);
+    return true;
+  }();
+  static_cast<void>(REGISTERED);
+}
+
+void populateDefaultQCOOptimizationPipeline(OpPassManager& pm) {
+  pm.addPass(qco::createMergeSingleQubitRotationGates());
+}
+
+LogicalResult runPassPipeline(ModuleOp module, const StringRef pipeline,
```

Cppcheck (2.21.0)

[style] The function 'runPassPipeline' is never used.

(unusedFunction)